plugin/trino-openlineage/src/test/java/io/trino/plugin/openlineage/TrinoEventData.java

```diff
@@ -58,22 +57,22 @@ private TrinoEventData()
         queryContext = new QueryContext(
                 "user",
                 "originalUser",
-                Set.of(),
+                Set.of("originalRoles"),
                 Optional.of("principal"),
-                Set.of(), // enabledRoles
-                Set.of(), // groups
-                Optional.empty(), // traceToken
-                Optional.empty(), // remoteClientAddress
-                Optional.empty(), // userAgent
-                Optional.empty(), // clientInfo
-                new HashSet<>(), // clientTags
-                new HashSet<>(), // clientCapabilities
+                Set.of("enabledRoles"),
+                Set.of("groups"),
+                Optional.of("traceToken"),
+                Optional.of("remoteClientAddress"),
+                Optional.of("userAgent"),
+                Optional.of("clientInfo"),
+                Set.of("clientTags"),
+                Set.of("clientCapabilities"),
```

Comment:
Do we need to change all the default values ?

Reply:
Reverted, but I don't see any issues with this change